src/App.js

```diff
@@ -430,19 +429,17 @@ class App extends React.Component {
                           permissions={permissions}
                         >
                           <div css="position: relative; z-index: 0">
-                            <Wrapper
+                            <OrgView
                               visible={mode === APP_MODE_ORG}
```

Comment:
```suggestion
                              activeInstanceId={locator.instanceId}
                              visible={mode === APP_MODE_ORG}
```

The menu item highlights weren't showing, I think we need to pass `activeInstanceId` here.